gunicorn/tests/test_check.py

```diff
@@ -10,7 +10,9 @@
 
 
 def _assert_metrics(aggregator):
-    aggregator.assert_metric("gunicorn.workers", tags=['app:dd-test-gunicorn', 'state:idle'], value=4, count=1)
+    # We don't check the value of gunicorn.workers. We were getting flaky test failures when Gunicorn didn't spin up
+    # all the requested (4) workers fast enough for them to show up in the metric.
+    aggregator.assert_metric("gunicorn.workers", tags=['app:dd-test-gunicorn', 'state:idle'], count=1)
     aggregator.assert_metric("gunicorn.workers", tags=['app:dd-test-gunicorn', 'state:working'], value=0, count=1)
```

Comment:
you should also remove the `value=0` there too, otherwise the test would fail

Reply:
I'm not sure about that. iiuc our setup just spins up gunicorn, there are always `0` workers with `state:working`. But maybe I'm missing something in our setup.

Reply:
Oh, I see. Then I will indeed remove the `0`, thanks!